frontend/src/container/FormAlertRules/ChQuerySection/transform.ts

```diff
@@ -1,39 +0,0 @@
-import { IChQuery } from 'types/api/alerts/compositeQuery';
-import { IClickHouseQuery } from 'types/api/dashboard/getAll';
-
-// @description rawQueryToIChQuery transforms raw query (from ClickHouseQueryBuilder)
-// to alert specific IChQuery format
-export const rawQueryToIChQuery = (
-	src: IChQuery,
-	rawQuery: string | undefined,
-	legend: string | undefined,
-	toggleDelete: boolean | undefined,
-): IChQuery => {
-	if (toggleDelete) {
-		return {
-			rawQuery: '',
-			legend: '',
-			name: 'A',
-			disabled: false,
-			query: '',
-		};
-	}
-
-	return {
-		rawQuery: rawQuery !== undefined ? rawQuery : src.query,
-		query: rawQuery !== undefined ? rawQuery : src.query,
-		legend: legend !== undefined ? legend : src.legend,
-		name: 'A',
-		disabled: false,
-	};
```

Comment:
Where is the equivalent of this in the new code? @palashgdev When such significant changes are getting merged, we need to ensure we do it carefully. Reviewing the functionality in the code (not from the UI testing) is important.